modules/openapi-generator/src/test/resources/3_0/python-experimental/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml

```diff
@@ -1097,6 +1097,19 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HealthCheckResult'
+  /farm/animals:
+    get:
+      tags:
+        - farm
```

Comment:
This tag has generated a new api which adds another file.
Can you change it to one of the existing tags like `pet` or `fake`
That way a new file will not be generated

Reply:
I would prefer to have it in separate API.